contracts/access/manager/IAccessManager.sol

```diff
@@ -29,6 +29,13 @@ interface IAccessManager {
     event OperationCanceled(bytes32 indexed operationId, uint32 indexed nonce);
 
     event RoleLabel(uint64 indexed roleId, string label);
+    /**
+     * @dev Emitted when `account` is granted `roleId`.
+     *
+     * NOTE: The meaning of the `since` argument depends on the `newMember` argument.
+     * If the role is granted to a new member, the `since` argument indicates when the delay goes into effect,
+     * otherwise it indicates the time when the roleId comes into effect.
```

Comment:
```suggestion
     * If the role is granted to a new member, the `since` argument indicates when the account gets the role,
     * otherwise it indicates the execution delay for this account and roleId is updated.
```